Bottom: f3bfe9259def378dfdf63838c7b605ce77190249
Top:    04d575518b15337c20d20d1731d796dd81bd05c3
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-22 13:48:30 -0500

Refresh of sshd-create-role

---
diff --git a/roles/setup-remote-user/tasks/main.yml b/roles/setup-remote-user/tasks/main.yml
index ed59a08..a4b3c79 100644
--- a/roles/setup-remote-user/tasks/main.yml
+++ b/roles/setup-remote-user/tasks/main.yml
@@ -32,24 +32,3 @@
     line: 'ansibleremote ALL=(ALL) NOPASSWD: ALL'
     regexp: 'ansibleremote ALL=(ALL) NOPASSWD: ALL'
     state: present
-
-- name: lockdown root user
-  lineinfile:
-    dest: /etc/ssh/sshd_config
-    regexp: '^[#]?PermitRootLogin .*$'
-    line: 'PermitRootLogin no'
-    state: present
-
-- name: Set ssh ListenAddress
-  lineinfile:
-    dest: /etc/ssh/sshd_config
-    regexp: '^[#]ListenAddress .*$'
-    replace: 'ListenAddress {{ ansible_br0["ipv4"]["address"] }}'
-  when: 
-    - ansible_os_family == "RedHat"
-    - ansible_br0 is defined and ansible_br0.ipv4 is defined and ansible_br0.ipv4.address is defined
-
-- name: restart sshd service
-  service:
-    name: sshd
-    state: restarted
